src/NUnit.Xml.TestLogger/NUnitXmlSerializer.cs

```diff
@@ -242,7 +244,7 @@ private static XElement CreateTestCaseElement(TestResultInfo result)
         {
             var element = new XElement(
                 "test-case",
-                new XAttribute("name", result.TestCase.DisplayName),
+                new XAttribute("name", result.Method),
```

Comment:
We should use `result.TestCaseDisplayName` to avoid regressions.

Reply:
If we use the display name, we are giving up the benefit of the adapter specific code in the core logger. At is stands, only the junit logger is using the method right now. But I thought that was the goal of the core logger.

I agree this could be a breaking change, but it seems like the right direction.

Reply:
Well, maybe this issue is going to be important. https://github.com/spekt/junit.testlogger/issues/57